org.uqbar.project.wollok.typeSystem.xsemantics.ui/src/org/uqbar/project/wollok/typeSystem/xsemantics/ui/labeling/XSemanticsWollokTypeSystemLabelExtension.xtend

```diff
@@ -14,16 +16,16 @@ class XSemanticsWollokTypeSystemLabelExtension implements WollokTypeSystemLabelE
 	val Logger log = Logger.getLogger(this.class)
 
 	override resolvedType(EObject o) {
-		// if disabeld
-		if (!WollokTypeSystemActivator.^default.isTypeSystemEnabled(o))
+		// if disabled
+		if (!WollokTypeSystemActivator.^default.isTypeSystemEnabled(o.project))
```

Comment:
TS configuration should be per-project, it does not make sense to have more granularity